application-templates/typescript/connect.yaml

```diff
@@ -54,4 +54,15 @@ deployAs:
         - key: CTP_CLIENT_SECRET
           description: commercetools Composable Commerce client secret
         - key: CTP_SCOPE
-          description: commercetools Composable Commerce client scope
\ No newline at end of file
+          description: commercetools Composable Commerce client scope
+  - name: mc-app
+    applicationType: mc-app
+    endpoint: /mc-app
```

Comment:
@dario-commercetools  can we keep endpoint as `/` to avoid confusion between this and entryPointUriPath in MC app

Reply:
@prateek-ct done :)